packages/cozy-flags/src/index.js

```diff
@@ -7,8 +7,20 @@ const flag = isNode
   ? require('./node/flag').default
   : require('./browser/flag').default
 
-if (__ENABLED_FLAGS__ && Array.isArray(__ENABLED_FLAGS__)) {
-  __ENABLED_FLAGS__.forEach(enabledFlag => flag(enabledFlag, true))
+/**
+ * Enables a list of flags
+ * @param {string[]} flagsToEnable
+ */
+export function enableFlags(flagsToEnable) {
+  if (!Array.isArray(flagsToEnable)) {
+    return
+  }
+
+  flagsToEnable.forEach(flagToEnable => flag(flagToEnable, true))
+}
```

Comment:
Could `enable` be attached on the exported `flag` function ? This enables namespacing for free `flags.enable` instead of `enableFlags`.